app/assets/stylesheets/application.css.scss

```diff
@@ -229,6 +229,11 @@ h5 {
    margin-top:25px;
 }
 
+.info-form .checkbox label {
```

Comment:
Just want to confirm that declaring a form style is necessary here -- can we get away without the additional custom class and styling?